analyses/sample-distribution-analysis/04-ns-for-manuscript.Rmd

```diff
@@ -0,0 +1,113 @@
+---
+title: "Sample numbers for manuscript"
+author: "Jo Lynne Rokita for D3b"
+date: "2022"
+output: 
+  html_notebook:
+    toc: TRUE
+    toc_float: TRUE
+params:
+  release: release-v21-20210820
+---
+
+```{r setup, include=FALSE}
+library(tidyverse)
+```
+
+## Set directories 
+```{r set directories}
+root_dir <- rprojroot::find_root(rprojroot::has_dir(".git"))
+data_dir <- file.path(root_dir, "data")
+analysis_dir <- file.path(root_dir, "analyses", "sample-distribution-analysis")
+```
+
+## Read in Files
+```{r read in files}
+hist_file <- file.path(data_dir, "pbta-histologies.tsv")
+cell_lines_file <- file.path(root_dir, "analyses", "molecular-subtyping-HGG", "input", "cell-line-composition.tsv")
```

Comment:
Do we know if this file is up-to-date for v21? Here: https://github.com/AlexsLemonade/OpenPBTA-analysis/blob/master/analyses/molecular-subtyping-HGG/input/cell-line-composition.tsv